controller/static/app/containers/containers.controller.js

```diff
@@ -227,6 +254,8 @@
 
         function containerStatusText(container) {
             if(container.Status.indexOf("Up")==0){
+                if (container.Status.indexOf("(Paused)") != -1)
```

Comment:
Ok. But another question ,I found you are using ```container.state.Runing```  to do this branch checking ,can we make this for constancy?